e2e-tests/cypress/tests/support/api/bots.ts

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {Bot, BotPatch} from '@mattermost/types/bots';
+import {getRandomId} from '../../utils';
+import {ChainableT} from 'tests/types';
+
+// *****************************************************************************
+// Bots
+// https://api.mattermost.com/#tag/bots
+// *****************************************************************************
+
+/**
+ * Create a bot.
+ * See https://api.mattermost.com/#tag/bots/paths/~1bots/post
+ * @param {string} options.bot - predefined `bot` object instead of random bot
+ * @param {string} options.prefix - 'bot' (default) or any prefix to easily identify a bot
+ * @returns {Bot} out.bot: `Bot` object
+ *
+ * @example
+ *   cy.apiCreateBot().then(({bot}) => {
+ *       // do something with bot
+ *   });
+ */
+function apiCreateBot(options?: {prefix: string; bot?: BotPatch}): ChainableT<{bot: Bot & {fullDisplayName: string}}> {
```

Comment:
There were tests failing with the following error:
<img width="934" alt="Screenshot 2024-10-11 at 11 35 40 AM" src="https://github.com/user-attachments/assets/318cbd01-37be-4f60-9548-e5bf29394776">

Reply:
I have made some changes, could you please give me some feedback?
Thanks in advance.